hoomd/filter/tags.py

```diff
@@ -1,21 +1,34 @@
-from hoomd.filter.filter_ import _ParticleFilter
+"""Define the Tags filter."""
+
+from hoomd.filter.filter_ import ParticleFilter
 from hoomd._hoomd import ParticleFilterTags
 import numpy as np
 
 
-class Tags(_ParticleFilter, ParticleFilterTags):
+class Tags(ParticleFilter, ParticleFilterTags):
+    """Select particles by tag.
```

Comment:
It may be worth going into what a particle tag is briefly, and how to get it from the state.

Reply:
Done.